Lib/argparse.py

```diff
@@ -1971,10 +1971,9 @@ def take_action(action, argument_strings, option_string=None):
                 action(self, namespace, argument_values, option_string)
 
         # function to convert arg_strings into an optional action
-        def consume_optional(start_index):
+        def consume_optional(start_index, option_tuple):
 
             # get the optional identified at this index
```

Comment:
This also needs to be updated.

Is `option_tuple` even used? I'm not sure what the standard is for these situations but you could also consider just passing in the values directly.

Reply:
Unpacking `option_tuple` seems like a good alternative.